deployments/kubernetes/chart/ingressmonitorcontroller/templates/imc.yaml

```diff
@@ -0,0 +1,175 @@
+apiVersion: v1
```

Comment:
The chart should have separate resources instead of having a single jumbo file. We have a mechanism implemented in pipeline-library which renders the whole chart and packages it into a single file and pushes it. So I think this should be reverted and the newer version of pipeline-library should be used for generating a single file